test/functionalTest/testHarness.sh

```diff
@@ -1131,7 +1131,7 @@ do
     continue;
   fi
 
-  if [ $toIx != 0 ] && [ $testNo -gt $toIx ]
+  if [ $toIx == "end" ] || [[ $toIx != 0  &&  $testNo -gt $toIx ]]
```

Comment:
Should it not be `$toIx !=  "end" ]` ?  (don't skip if $toIx == "end")

Reply:
No, this is not going to work.
toIx is an integer variable (`typeset -i toIx`).
We will need another mechanism.
I'd propose to check for 'end' in a layer before testHarness.sh is executed ans set the end to 100000

Reply:
Fixed in a83a3d7